Controller/RegistrationFOSUser1Controller.php

```diff
@@ -56,10 +56,10 @@ public function registerAction()
                 $url = $this->generateUrl('fos_user_registration_check_email');
             } else {
                 $authUser = true;
-                $route = $this->get('session')->get('sonata_basket_delivery_redirect');
+                $route = $this->get('session')->get('sonata_user_redirect_url');
 
                 if (null !== $route) {
-                    $this->get('session')->remove('sonata_basket_delivery_redirect');
+                    $this->get('session')->remove('sonata_user_redirect_url');
```

Comment:
It looks indeed weird… It was introduced in 4dbf95fc5a103eaa364abfd3ce3d805e5f7dcda8